Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlayDrawableJoystick.java

```diff
@@ -93,6 +96,7 @@ public void TrackEvent(MotionEvent event)
 			if (getBounds().contains((int)event.getX(pointerIndex), (int)event.getY(pointerIndex)))
 			{
 				mPressedState = true;
+				getVirtBounds().offset((int)event.getX(pointerIndex) - getVirtBounds().centerX(), (int)event.getY(pointerIndex) - getVirtBounds().centerY());
```

Comment:
Is the mVirtBounds object used outside of this class? If not, you could just store the x/y coordinate here and use them again in line 128 / 130 instead of getVirtBounds().centerX/Y.